### ISSUE TITLE
Comment utiliser des dés évolutifs dans les attaques ?

### ISSUE THREAD

**Issue by Ulty** [stephaned68/COF2e#3]
J'ai peut-être raté quelque chose dans la documentation, mais je n'ai pas trouvé comment utiliser des dés évolutifs.

**Comment by stephaned68** [comment]
Si tu mets `1d4°` dans les dés de DM ou dans la rubrique "Spécial", ça doit faire la substitution par `1@{devol}` avant de faire le jet dans le chat. `1d4°` ou `2d4°` ou whatever. La substitution est également faite avant d'envoyer la description d'une capacité dans le chat (bouton bulle d'aide).

**Comment by Ulty** [comment]
C'est pas très pratique de devoir taper un °. Il pourrait peut-être y avoir une syntaxe alternative ?

**Comment by stephaned68** [comment]
C'est indiqué comme ça dans le bouquin de règles, alors je me suis dit que pour ceux qui n'utilisent pas l'export Chroniques Mobiles pour importer un JSON mais font du copier-coller depuis le PDF, ça marcherait tout seul. 
Tu penses à quelque chose en particulier ? `d@` ? `dE` ?

**Comment by Ulty** [comment]
Oui, c'est de toutes façons nécessaire de reconnaître le °, tu as raison. Pour un truc plus facile à taper, il faut que ça n'interfère pas avec les syntaxes de macros ou d'expressions de dés. Donc `dE`, ça me semblerait pas mal.

**Comment by Ulty** [comment]
J'ai l'impression que `dE` n'est pas reconnu. `d4°` fonctionne correctement, en revanche.

**Comment by Ulty** [comment]
Si tu veux bien me mettre comme collaborateur, je pourrais rouvrir les issues. Sinon, je peux aussi en ouvrir une nouvelle en mentionnant celle-ci.

**Comment by Ulty** [comment]
Merci !

**Comment by stephaned68** [comment]
Je t'ai ajouté en collaborateur
J'avais bien ajouté dE dans les RegExp qui créent les inlines (description des capacités, champ _spécial_ des attaques) mais pas dans le bouton de jet d'attaque
